src/pages/NIHicWebform.js

```diff
@@ -0,0 +1,2266 @@
+          };
+        });
+        callback(options);
+      });
+  };
+
+  setPublicAccess = (value) => {
+    this.setState(prev => {
+      prev.datasetData.publicAccess = value;
+      return prev;
+    });
+  }
+
+  setGeneralUse = () => {
+    this.setState(prev => {
+      prev.formData.generalUse = true;
+      prev.formData.hmb = false;
+      prev.formData.diseases = false;
+      prev.formData.ontologies = [];
+      return prev;
+    });
+  }
+
+  setHmb = () => {
+    this.setState(prev => {
+      prev.formData.generalUse = false;
+      prev.formData.hmb = true;
+      prev.formData.diseases = false;
+      prev.formData.ontologies = [];
+      return prev;
+    });
+  };
+
+  setPoa = () => {
+    this.setState(prev => {
+      prev.formData.generalUse = false;
+      prev.formData.hmb = false;
+      prev.formData.poa = true;
+      prev.formData.diseases = false;
+      prev.formData.ontologies = [];
+      return prev;
+    });
+  };
+
+  setDiseases = () => {
+    this.setState(prev => {
+      prev.formData.generalUse = false;
+      prev.formData.hmb = false;
+      prev.formData.diseases = true;
+      return prev;
+    });
+  };
+
+  onOntologiesChange = (data) => {
+    this.setState(prev => {
+      prev.formData.ontologies = data || [];
+      return prev;
+    });
+  };
+
+  setOther = () => {
+    this.setState(prev => {
+      prev.formData.generalUse = false;
+      prev.formData.hmb = false;
+      prev.formData.diseases = false;
+      prev.formData.other = true;
+      prev.formData.ontologies = [];
+      return prev;
+    });
+  };
+
+  setOtherText = (e) => {
+    const value = e.target.value;
+    this.setState(prev => {
+      prev.formData.other = true;
+      prev.formData.otherText = value;
+      return prev;
+    });
+  };
+
+  back = (e) => {
+    this.props.history.goBack();
+  };
+
+  formatOntologyItems = (ontologies) => {
+    return ontologies.map((ontology) => {
+      return {
+        id: ontology.id || ontology.item.id,
+        key: ontology.id || ontology.item.id,
+        value: ontology.id || ontology.item.id,
+        label: ontology.label || ontology.item.label,
+        definition: ontology.definition || ontology.item.definition,
+        item: ontology || ontology.item
+      };
+    });
+  };
+
+  render() {
+
+    const controlLabelStyle = {
+      fontWeight: 500,
+      marginBottom: 0
+    };
+
+    const {
+      hmb = false,
+      poa = false,
+      diseases = false,
+      other = false,
+      otherText = '',
+      genetic = false,
+      forProfit = false,
+      publication = false,
+      collaboration = false,
+      ethics = false,
+      geographic = false,
+      moratorium = false,
+      methods = false,
+      generalUse = false,
+    } = this.state.formData;
+    const ontologies = this.formatOntologyItems(this.state.formData.ontologies);
+    const publicAccess = false;
+    const { npoa } = !poa;
+    const { showValidationMessages } = this.state;
```

Comment:
I don't think we're using this any more ... it should be removed from here and state.